Code/Framework/AzQtComponents/AzQtComponents/Components/DockBarButton.cpp

```diff
@@ -15,6 +15,7 @@
 #include <QStyleOptionToolButton>
 #include <QStylePainter>
 #include <QTimer>
+#include <QEvent>
```

Comment:
Uber nitpick: Could put `QEvent` just below `QApplication` for alphabetical ordering